BaseFilter/sections/specific.txt

```diff
@@ -275,6 +275,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+||*.xyz^$domain=xhamster16.*|hamsterix.*|xhamster13.*|xhamster18.*|xhamster19.*|xhamster11.*|xhamster5.*|xhamster10.*|xhamster4.*|xhamster3.*|xhamster2.*|xhamster.*
```

Comment:
Why not `||xyz^$domain=` ? Also I think better to limit to subdocument.

Reply:
Are all mentioned request advertisement-related?

Reply:
Hm. I don't see requests to `.xyz`

Reply:
Please click one of videos.
@Alex-302

Reply:
> Are all mentioned request advertisement-related?

Yes.

Reply:
Pushed.